json/silverbullet.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Silverbullet",
+    "slug": "silverbullet",
+    "categories": [
+        14
+    ],
+    "date_created": "2024-12-03",
+    "type": "ct",
+    "updateable": true,
+    "privileged": false,
+    "interface_port": 3000,
+    "documentation": null,
```

Comment:
No docu?

Reply:
I checked a bunch of other JSONs and all had `documentation: null`, thought the field might be unused.
Guess I was unlucky.
Either way, the website is the documentation so I linked a sub-page, that seemed fitting.